### DISCUSSION TITLE
Simple webhook notifications

### DISCUSSION THREAD

**Discussion by stephenrjr** [alexjustesen/speedtest-tracker#869] [Ideas]
Would love to see a light-weight webhook notification added for more modest alerting needs.

As an example, I use Synology Chat and have setup several incoming webhooks to notify specific chat channels with topical information.  Here is an explanation of the webhook interface from Synology:

[Using Integration in Synology Chat](https://kb.synology.com/en-af/DSM/help/Chat/chat_integration?version=7)

**Comment by Bergruebe** [comment]
I would also like to see that. In my use case, I would connect it to my Uptime Kuma, so I have all up and down time in one place.

So it would be great, if it would be possible to add the latest ping or up-/download via a variable to the web hook.

**Comment by kinouzero** [comment]
Same, there isn't Slack integration so webhook can do the trick to send to Slack.

**Comment by alexjustesen** [comment]
Attaching https://github.com/alexjustesen/speedtest-tracker/pull/754 for lineage.

**Comment by rusty1281** [comment]
Just tested the webhook with Mattermost, but while the test is OK, there is still no payload delivered. Any news on the general webhook integration?

tnx!
